moveit_experimental/collision_distance_field/CMakeLists.txt

```diff
@@ -28,3 +23,11 @@ endif()
 add_library(collision_detector_hybrid_plugin src/collision_detector_hybrid_plugin_loader.cpp)
 target_link_libraries(collision_detector_hybrid_plugin ${catkin_LIBRARIES} ${MOVEIT_LIB_NAME})
 
+install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_hybrid_plugin
+  LIBRARY DESTINATION lib)
+  
+install(DIRECTORY include/
+  DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

Comment:
I just used the standard ones from the ros examples... I thought maybe that
is more cleaner....

On Sep 7, 2016 4:48 PM, "Jorge Nicho" <notifications@github.com> wrote:

> In moveit_experimental/collision_distance_field/CMakeLists.txt
> <https://github.com/ros-planning/moveit/pull/155#discussion_r77837216>:
>
> > @@ -28,3 +23,11 @@ endif()
> >  add_library(collision_detector_hybrid_plugin src/collision_detector_hybrid_plugin_loader.cpp)
> >  target_link_libraries(collision_detector_hybrid_plugin ${catkin_LIBRARIES} ${MOVEIT_LIB_NAME})
> >
> > +install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_hybrid_plugin
> > +  LIBRARY DESTINATION lib)
> > +
> > +install(DIRECTORY include/
> > +  DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
>
> Do you know where the ${CATKIN_GLOBAL_INCLUDE_DESTINATION} place the
> header files? why not use just "include" instead?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros-planning/moveit/pull/155/files/08f205677c3109c30b8aafc6214d1e7cb2eaf98b..abbd8d3d9588c750d43890f95ec2a24930521e90#r77837216>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGuXcOpmkD1Ectpxm0P_wWz8OcQLCTcdks5qns7BgaJpZM4Js5U5>
> .
>